src/components/pages/Bridge/SelectAsset.vue

```diff
@@ -33,12 +33,11 @@ import SelectAssetMixin from '@/components/mixins/SelectAssetMixin';
 import TranslationMixin from '@/components/mixins/TranslationMixin';
 import { Components, ObjectInit } from '@/consts';
 import { lazyComponent } from '@/router';
-import type { BridgeAccountAsset } from '@/store/assets/types';
+import type { BridgeRegisteredAsset } from '@/store/assets/types';
```

Comment:
please don't use `@` for types imports cuz it breaks js lint sorting rules

Reply:
Will try to fix this in another PR (because of a lot of changes after linting)